rpcs3/Emu/Cell/lv2/sys_time.cpp

```diff
@@ -181,9 +183,15 @@ error_code sys_time_get_timezone(vm::ptr<s32> timezone, vm::ptr<s32> summertime)
 {
 	sys_time.warning("sys_time_get_timezone(timezone=*0x%x, summertime=*0x%x)", timezone, summertime);
 
-	*timezone   = 180;
-	*summertime = 0;
+#ifdef _WIN32
+	TIME_ZONE_INFORMATION tz{};
+	const DWORD tztype = GetTimeZoneInformation(&tz);
+	*timezone = tz.Bias;
+#else
+	*timezone = ::narrow<s32>(::timezone / 60);
+#endif
 
+	*summertime = 0;
```

Comment:
you can probably use tztype to get the summertime info:
![image](https://user-images.githubusercontent.com/23019877/116131140-4f5d8400-a6cc-11eb-9ece-25cbfbc699e6.png)

Reply:
I'm not sure how its value is interpreted. Easier to keep it zero.

Reply:
then please add a TODO comment